docs/index.rst

```diff
@@ -46,12 +52,12 @@ License: BSD-3
 News
```

Comment:
Ok, no problem -- 'What's New' is fine, but I don't really think we need to put things like "April 28, 2015: astroplan repo set up" in the docs...